docker/datahub-actions/env/docker.env

```diff
@@ -1,4 +1,6 @@
+DATAHUB_GMS_HOST=datahub-gms
```

Comment:
Wait wait wait - Don't we already have GMS_HOST  and GMS_PORT? 

Why are we duplicating information we also have!

Reply:
I am going to deprecate these. To be consistent with rest of the services.